### DISCUSSION TITLE
(Moved discussion from elsewhere)

### DISCUSSION THREAD

**Discussion by ericsink** [ericsink/SQLitePCL.raw#551] [General]
> What I mean below was not working..
SQLitePCL.raw.SetProvider(New SQLitePCL.SQLite3Provider_sqlite3())

> SQLite3Provider_sqlite3 this method seems to be missing from the latest SqlitePCLRaw.bundle.e_sqlcipher package I am using. It is giving compile error.

@vissharm

Yes, provider_sqlite3 is not in bundle e_sqlcipher.  Because:

The SQLitePCLRaw provider for "sqlite3" is used to reference a SQLite library named "sqlite3", and my unofficial/unsupported SQLCipher builds are not named that. They are named "e_sqlcipher", specifically to avoid naming conflicts with other builds, such as, for example, the official ones from Zetetic.

If you need this provider, you can add this nuget package:

https://www.nuget.org/packages/SQLitePCLRaw.provider.sqlite3

**Comment by vissharm** [comment]
Thanks, will try that.
